.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,210 @@
+
+> Short and descriptive example bug report title
+>
+> A summary of the issue and the browser/OS environment in which it occurs. If
+> suitable, include the steps required to reproduce the bug.
+>
+> 1. This is the first step
+> 2. This is the second step
+> 3. Further steps, etc.
+>
+> `<url>` - a link to branch with the reduced test case
+>
+> Any other information you want to share that is relevant to the issue being
+> reported. This might include the lines of code that you have identified as
+> causing the bug, and potential solutions (and your opinions on their
+> merits).
+
+&nbsp;
+
+
+### Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the `over_react` team of the merits of this feature. Please provide as much detail and context as possible.
+
+&nbsp;
+
+
+### Pull requests
+
+Good pull requests - patches, improvements, new features - are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
+
+__Please ask first__ before embarking on any significant pull request (e.g. implementing features, refactoring code, porting to a different language), otherwise you risk spending a lot of time working on something that the project's lead developers might not want to merge into the project.
+
+Please adhere to the [Dart Style Guide][dart-style-guide] for all changes contained in your pull requests.
+
+Adhering to the following process is the best way to get your work included in the project:
+
+1. [Fork](http://help.github.com/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
+
+   ```bash
+   # Navigate to the directory where you store repos locally
+   cd ~/your-local-git-repo-spot
+   # Clone your fork of the repo into the current directory
+   git clone git@github.com:<your-username>/over_react
+   # Navigate to the newly cloned directory
+   cd ~/your-local-git-repo-spot/over_react
+   # Assign the repo you forked from to a remote called "upstream"
+   git remote add upstream git@github.com:Workiva/over_react
+   ```
+
+2. If you cloned a while ago, get the latest changes from upstream:
+
+   ```bash
+   git checkout master
+   git pull upstream master
+   ```
+
+3. Create a new topic branch that will contain your feature, change, or fix:
+
+   ```bash
+   git checkout -b <topic-branch-name>
+   ```
+
+4. Commit your changes in logical chunks. Please adhere to these [git commit message guidelines](#git-commit-message-standards) or your code is unlikely be merged into the master branch. Optionally, you can use Git's [interactive rebase](https://help.github.com/articles/interactive-rebase) feature to tidy up your commits before making them public.
+
+5. Write tests for your changes.  
+  1. There are no exceptions.  
+  2. If your PR doesn't have accompanying test coverage - your code 
+     will not get merged.
```

Comment:
This wording isn't super friendly. Not saying we shouldn't require changes to be tested, but may be good to include something like: "if you're having trouble, reach out in your PR about how to best go about testing your changes."